### PR TITLE
allow passing certs_tar for Katello app

### PR DESCRIPTION
**Pull request by sean797**
generate certs_tar with https://github.com/Katello/katello-installer/pull/628

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -28,6 +28,8 @@
 #
 # $repo_export_dir::    Directory to create for repository exports
 #
+# $certs_tar::                          Path to a tar with certs for the node
```
Comment (ekohl) [inline manifests/init.pp]: I put the manage flags under the advanced parameters and I think this is also an advanced feature.

**Review by ekohl — COMMENTED** [review]
Neat. I hadn't thought about this.

**Comment by sean797** [comment]
Updated, if there is anything else we can change it & review in the other PR with a wider audience :+1:

**Comment by ekohl** [comment]
I like the general idea and hadn't thought of this but it made me wonder if this is something we could actually do in puppet-certs. That should allow us to remove it from puppet-foreman_proxy_content and probably implement it in the right place.

I'd be happy to update the PRs to include this so we can at least get a demo setup working.

**Comment by sean797** [comment]
> I like the general idea and hadn't thought of this but it made me wonder if this is something we could actually do in puppet-certs. 

That would probably be a good idea :+1: 

Anyway, let's merge, get demos working and discuss in the real PR ;)

**Pull request merged**